projects/cas3-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/approvedpremesis/EventDetails.kt

```diff
@@ -48,7 +48,13 @@ data class BookingProvisional(
     val notes: String
 ) : Cas3Event {
     override val urn = "urn:hmpps:cas3:booking-provisionally-made:$bookingId"
-    override val noteText = "${DeliusDateFormatter.format(expectedArrivedAt)} $notes $bookingUrl"
+    override val noteText =
+        listOfNotNull(
+            "Expected arrival date:",
+            DeliusDateFormatter.format(expectedArrivedAt),
+            notes,
+            bookingUrl
+        ).joinToString(" ")
```

Comment:
I wonder if notes and bookingUrl should be on different lines, rather than just space-separating them?  When NDST raised it a little while ago it looked a bit odd e.g.
```
Expected arrival date: 01/01/2000 some notes https://....
```

vs
```
Expected arrival date: 01/01/2000 
some notes

https://....
```